Editor/Window/ManagedEC2Page.cs

```diff
@@ -151,12 +152,20 @@ private void UpdateGUI()
             }
             else if (stackStatus.IsStackStatusFailed())
             {
-                _statusIndicator.Set(State.Failed, textProvider.Get(Strings.ManagedEC2DeployStatusFailed));
+                var errorText = textProvider.Get(Strings.ManagedEC2DeployStatusFailed);
+                _statusIndicator.Set(State.Failed, errorText);
+                _deployStatusBox.Show(StatusBox.StatusBoxType.Error,  errorText, null, string.Format(Urls.AwsCloudFormationTemplate, _stateManager.Region, _deploymentSettings.CurrentStackInfo.StackId), Strings.LabelDeploymentCloudFormationConsole);
```

Comment:
It appears that in the mocks, there should be "learn more" style link next to the status indicator. It should read "View in CloudFormation Console", it links to the stack info like right here, and should display for every status other than "Not Deployed".

With that link always being present, we can remove the link from both of these status boxes.

Reply:
Fixed this